modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/applications/forms/steps/Review/index.tsx

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayTable from '@clayui/table';
+import React from 'react';
+
+const Review = () => {
+	const coverages = [
+		{
+			coverage: 'BODILY INJURY',
```

Comment:
Please, change the property name.
```suggestion
			name: 'BODILY INJURY',
```

Reply:
done. tks